includes/Abilities/Utilities/Utilities.php

```diff
@@ -0,0 +1,404 @@
+<?php
+/**
+ * Utility WordPress Abilities implementation.
+ *
+ * @package WordPress\AI
+ */
+
+declare( strict_types=1 );
+
+namespace WordPress\AI\Abilities\Utilities;
+
+use WP_Error;
+
+/**
+ * Utility WordPress Abilities.
+ *
+ * @since 0.1.0
+ */
+class Utilities {
+
+	/**
+	 * Register any needed hooks.
+	 *
+	 * @since 0.1.0
+	 */
+	public function register(): void {
+		add_action( 'wp_abilities_api_init', array( $this, 'register_abilities' ) );
+	}
+
+	/**
+	 * Registers any needed abilities.
+	 *
+	 * @since 0.1.0
+	 */
+	public function register_abilities(): void {
+		$this->register_get_content_ability();
+		$this->register_get_title_ability();
+		$this->register_get_name_ability();
+		$this->register_get_author_ability();
+		$this->register_get_type_ability();
+		$this->register_get_excerpt_ability();
+		$this->register_get_terms_ability();
+	}
+
+	/**
+	 * Registers the get-content ability.
+	 *
+	 * @since 0.1.0
+	 */
+	private function register_get_content_ability(): void {
+		wp_register_ability(
+			'ai/get-content',
```

Comment:
```suggestion
			'ai/get-post-content',
```

Reply:
This ability no longer exists so resolving this